docs/esasky/esasky.rst

```diff
@@ -51,7 +51,10 @@ There are two query objects methods in this module
 almost the same way except that one has catalogs as input and output and the 
 other one has mission names and observations as input and output. 
 
-For catalogs, the query returns a maximum of 2000 sources per mission.
+For catalogs, the query returns a maximum of 2000 sources per mission by 
+default. However, this can be modified by the row_limit parameter.
+You can set the parameter to -1, which will result in the maximum number of
+sources (currently 100 000).
```

Comment:
Would setting it to, e.g., 999999999 also work, or would the server reject that?